### PR TITLE
HDDS-4156. add hierarchical layout to Chinese doc

### PR DESCRIPTION
**Pull request by flirmnave**
## What changes were proposed in this pull request?

Add hierarchical layout to Chinese doc

## What is the link to the Apache JIRA

https://issues.apache.org/jira/browse/HDDS-4156

## How was this patch tested?

Test on local use hugo server.
Screenshot from 2020-08-31 22-40-19

### CODE REVIEW

File: hadoop-hdds/docs/content/concept/_index.zh.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 title: 概念
 date: "2017-10-10"
 menu: main
-weight: 6
+weight: 3
```
Comment (iamabug) [inline hadoop-hdds/docs/content/concept/_index.zh.md:5]: Could you please explain the reason why `weight` is changed here? @flirmnave

**Comment by iamabug** [comment]
Thanks for the work @flirmnave .

According to what I see in localhost, it seems there are some subpages left.



Also, as for `Features` menu, I think it is both okay to finish it in a separate PR or in this one.

File: hadoop-hdds/docs/content/concept/_index.zh.md (hunk 1 above)
Comment (flirmnave) [reply hadoop-hdds/docs/content/concept/_index.zh.md:5]: Hi,
I see that it weight 3 in english version.
https://github.com/apache/hadoop-ozone/blame/master/hadoop-hdds/docs/content/concept/_index.md#L5
So I change it, Should I change it back?

**Comment by flirmnave** [comment]
> Thanks for the work @flirmnave .
> 
> According to what I see in localhost, it seems there are some subpages left.
> 
> 
> 
> Also, as for `Features` menu, I think it is both okay to finish it in a separate PR or in this one.


Thanks @iamabug review and comment.
Some of the documents have not been translated into Chinese
So some subpages left.

For `Features` menu, I have create a Jira issue.
https://issues.apache.org/jira/browse/HDDS-4184

File: hadoop-hdds/docs/content/concept/_index.zh.md (hunk 1 above)
Comment (iamabug) [reply hadoop-hdds/docs/content/concept/_index.zh.md:5]: No, it is the right thing to do, sorry for not noticing this.

Comment (flirmnave) [reply hadoop-hdds/docs/content/concept/_index.zh.md:5]: It’s ok.

**Comment by xiaoyuyao** [comment]
LGTM overall. Only have one question: should move the GDPR under the Security not Features. This may apply to the original EN document as well.

**Comment by flirmnave** [comment]
> LGTM overall. Only have one question: should move the GDPR under the Security not Features. This may apply to the original EN document as well.

Thanks @xiaoyuyao for the review and comment.
I create a Jira issue [HDDS-4258](https://issues.apache.org/jira/browse/HDDS-4258) for move the GDPR under the Security.

**Comment by cku328** [comment]
Overall LGTM. Thanks @flirmnave for working on this.
This PR will make the Chinese docs more friendly.

**Review by cku328 — APPROVED** [review]
Overall LGTM. Thanks @flirmnave for working on this.
This PR will make the Chinese docs more friendly.

**Comment by elek** [comment]
Comments from @xiaoyuyao are addressed (issue is created) and @cku328 also reviewed it (thanks for both)

I don't speak Mandarin well, but merging this now as structure change seems to be ok.

Thanks the contribution @flirmnave

**Comment by flirmnave** [comment]
Thanks for @iamabug @xiaoyuyao @cku328 reviewing and @elek merging.

**Pull request merged**